packages/jsii-dotnet-analyzers/src/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup>
+    <JsiiVersion>0.15.0</JsiiVersion>
```

Comment:
Any chance this is actually a generated file that shouldn't be checked in?

Reply:
Yes, removing them